source/default/_patterns/02-molecules/vue-widget/src/vue-clock/vue-clock.vue

```diff
@@ -34,31 +34,34 @@
   </div>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import Vue from 'vue';
+import { TimeInt, TimeStyles } from './types';
+
+export default Vue.extend({
   name: 'VueClock',
   data() {
     return {
       now: new Date(),
     };
   },
   computed: {
-    time() {
+    time(): TimeInt {
       return {
         seconds: this.now.getSeconds(),
         minutes: this.now.getMinutes(),
         hours: this.now.getHours(),
       };
     },
-    degrees() {
-      const { seconds, minutes, hours } = this.time;
+    degrees(): TimeInt {
+      const { seconds, minutes, hours }: TimeInt = this.time;
       return {
-        seconds: parseInt((seconds / 60) * 360, 10),
-        minutes: parseInt((minutes / 60 + seconds / 60 / 60) * 360, 10),
-        hours: parseInt((hours / 12 + minutes / 60 / 12) * 360, 10),
+        seconds: (seconds / 60) * 360,
```

Comment:
THAAAAAAAAANKS

Reply:
:+1: Love when type systems remind us what types functions expect to receive and return.